cookie-banner-rules-list.json

```diff
@@ -3715,9 +3629,9 @@
     },
     {
       "click": {
-        "optIn": ".nds-btn.dialog-actions-accept-btn.css-60b779.ex41m6f0.btn-primary-dark.btn-md",
-        "optOut": ".nds-btn.dialog-actions-decline-btn.css-60b779.ex41m6f0.btn-primary-dark.btn-md",
-        "presence": ".dialog-content-wrapper.dialog-appear-done.dialog-enter-done"
+        "optIn": ".dialog-actions-accept-btn",
+        "optOut": ".dialog-actions-decline-btn",
+        "presence": ".dialog-content-wrapper"
```

Comment:
Too generic, this looks like it targets any dialog. I can see a `cookie-dialog-base` class, can we target that? It's fine if the buttons are more generic as long as the presence selector is specific. That's because we only query the buttons as children of the presence dialog.